coins.json

```diff
@@ -9,8 +9,13 @@
         "curve": "secp256k1",
         "publicKeyType": "secp256k1Extended",
         "explorer": "https://explorer.nebulas.io/#/tx/",
-        "url": "https://nebulas.io",
-        "rpcNodeInfo": "https://github.com/nebulasio/go-nebulas"
+        "info": {
+            "url": "https://nebulas.io",
+            "node": "https://github.com/nebulasio/go-nebulas",
```

Comment:
`client` may be more intuitive? That's what I've seen people refer to geth and parity as.